portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <style>
+    a    {
+        color: black;
+        text-shadow: 2px 2px 5px turquoise, 1px 1px 3px purple, 1px 2px 3px snow;
+        text-decoration: underline;
+    }
+    body {
+        background: url(images/better.jpg) no-repeat center center fixed;
+    }
+
+    html {
+        scroll-behavior: smooth;
+    }
+    </style>
+  </head>
+  <body>
+    <div id="content">
+        <div style="border-bottom:3px solid #000">
```

Comment:
This is allowed, but is bad practice. Adding the style attribute directly to the tag makes it separate from your main styling stuff (which you have up above) and so harder to track down and modify. It also doesnt allow it to be reused.

As a general rule, the goal of each of the languages should is:
HTML: This is the structure of my page/data
CSS: This is the visual way in which the page/data should be shown
JS: This is the way which my page should behave

And the goal is to keep things as much in their own domain as possible. So keeping all style attributes to instead use a stylesheet is preferable